CHANGELOG.md

```diff
@@ -11,6 +11,16 @@ project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 ### Fixed
 
+2.10.2
+==================
+### Fixed
+* Fix `Assertion failed: (object->InternalFieldCount() > 0), function Unwrap, file nan_object_wrap.h, line 32.` ([#2025](https://github.com/Automattic/node-canvas/issues/2025))
+### Added
+* Prebuilt binaries for Node.js v18
```

Comment:
More work has to be done for prebuilds (upgrading the base Debian image). Can you remove this line please?